app/scripts/core/directives/card.directive.js

```diff
@@ -216,7 +216,7 @@
                 var autoplay = false;
                 var cardVideo = findElement('.jw-card-controls')[0];
 
-                if (event.target === cardVideo) {
+                if (event.target === cardVideo || event.target.parentNode === cardVideo) {
```

Comment:
Maybe you could check if `event.target` is a child of `cardVideo`? That could cover a few more edge cases.

Reply:
True - good suggestoin